lib/checkother.cpp

```diff
@@ -1052,13 +1054,13 @@ void CheckOther::duplicateBreakError(const Token *tok, bool inconclusive)
     reportError(tok, Severity::style, "duplicateBreak",
                 "Consecutive return, break, continue, goto or throw statements are unnecessary.\n"
                 "Consecutive return, break, continue, goto or throw statements are unnecessary. "
-                "The second statement can never be executed, and so should be removed.", CWE(0U), inconclusive);
+                "The second statement can never be executed, and so should be removed.", CWE398, inconclusive);
```

Comment:
Yes, good catch. I fixed it on the same branch. 

I've added another commit to the branch, squashed the commits and updated the branch with a   git push -f .